src/components/TodoList/TodoList.tsx

```diff
@@ -1 +1,18 @@
-export const TodoList = () => {};
+import { TodoInfo } from '../TodoInfo';
+
+import { Todo } from '../../types/Todo';
+
+type Props = {
+  todos: Todo[],
```

Comment:
Consider renaming `todos` prop to `todoList` for better readability and consistency. 📚